yb-voyager/src/utils/utils.go

```diff
@@ -744,3 +744,13 @@ func CheckTools(tools ...string) []string {
 func BuildObjectName(schemaName, objName string) string {
 	return lo.Ternary(schemaName != "", schemaName+"."+objName, objName)
 }
+
+// ToTitleCase converts a snake_case string to a title case string with spaces.
+func ToTitleCase(snake string) string {
```

Comment:
would be good to mention `snake case` in the func name. So `SnakeCaseToTitleCase` ?